data_steward/tools/run_deid.py

```diff
@@ -252,26 +252,27 @@ def main(raw_args=None):
         if 'person_id' in field_names:
             parameter_list.append('--cluster')
 
-        LOGGER.info('Executing deid with:\n\tpython deid/aou.py %s',
-                    ' '.join(parameter_list))
+        LOGGER.info(
+            f"Executing deid with:\n\tpython deid/aou.py {' '.join(parameter_list)}"
+        )
 
         try:
             aou.main(parameter_list)
         except google.api_core.exceptions.GoogleAPIError:
             LOGGER.exception("Encountered deid exception:\n")
             exceptions.append(table)
         else:
-            LOGGER.info('Successfully executed deid on table: %s', table)
+            LOGGER.info(f"Successfully executed deid on table: {table}")
             successes.append(table)
 
     copy_suppressed_table_schemas(known_tables, args.input_dataset + '_deid')
 
-    LOGGER.info('Deid has finished.  Successfully executed on tables:  %s',
-                '\n'.join(successes))
+    LOGGER.info(
+        "Deid has finished.  Successfully executed on tables: {}".format(
+            '\n'.join(successes)))
```

Comment:
nit:  weird.  this changed to a formatted string instead of an f-string.

Reply:
Per above comment, this case caused me to discover a limitation of fstrings I wasn't aware of. Backslashes cannot be embedded into fstring expressions. My options here were:

- Use a format or substitution
- assign to a variable then fstring interpolate the variable

There's an argument for the latter instead, but I just went with this since apparently `.format` is still legal in this world. Let me know if you prefer the latter approach

Reply:
I think the `.format` is fine.  Adding this to a variable to allow f-string interpolation feels like too much extra work just for a logging statement.  I didn't realize there was a limitation with backslashes in f-strings either.  I just noticed the oddity and wondered why it existed.